docs/pages/variables.md

```diff
@@ -59,8 +59,10 @@ supplies a predefined variables with the SHA.
 | K8S\_PROBE\_INITIAL\_DELAY    | Seconds before health/ready checks starts           | 60                           |            |
 | K8S\_PROBE\_PERIOD            | How long between probe checks                       | 10                           |            |
 | K8S\_REPLICACOUNT             | Number of replicated Pods                           | 1                            |            |
-| K8S\_REQUEST\_CPU             | Request at least this much CPU (ex. 1000m)          |                              |            |
-| K8S\_REQUEST\_RAM             | Request at least this much RAM (ex. 512Mi)          |                              |            |
+| K8S\_REQUEST\_CPU             | Request at least this much CPU (ex. 1000m)          | 50m                          |            |
+| K8S\_REQUEST\_RAM             | Request at least this much RAM (ex. 512Mi)          | 128Mi                        |            |
+| K8S\_LIMITS\_CPU              | Limits max CPU (ex. 1000m)                          |                              |            |
```

Comment:
This should probably be `K8S_LIMIT_CPU` (same with RAM), note the plural S that should be removed.

Reply:
Fixed to singular. I probably thought keys in deployment manifest with naming.